src/main/java/stream/flarebot/flarebot/commands/moderation/mod/MuteCommand.java

```diff
@@ -18,28 +18,28 @@
     @Override
     public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message message, String[] args, Member member) {
         if (args.length == 0) {
-            MessageUtils.sendUsage(this, channel, sender);
+            MessageUtils.sendUsage(this, channel, sender, args);
         } else {
             User user = GeneralUtils.getUser(args[0], guild.getGuildId());
             if (user == null) {
                 MessageUtils.sendErrorMessage("Invalid user!!", channel);
                 return;
             }
             if (guild.getMutedRole() == null) {
-                MessageUtils.sendErrorMessage("Error getting the \"Muted\" role! Check FlareBot has permissions to create it!", channel);
+                MessageUtils.sendErrorMessage("Error getting the \"Muted\" role! Check FlareBot has permissions to create and apply it!", channel);
```

Comment:
Creating it yes we'll have a problem here, but we won't have a problem here with applying the role. we shouldn't give false info